.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Request a new format specification to be created'
+labels: spec-request
+---
+<!--
+A request for a format specification consist of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete the fields without data and all the comments from this template (including this one!).
+The resulting YAML blocks should contain valid YAML.
+-->
+```yaml
+# This block is the boilerplate of a KSY spec file. Must be a valid KSY. Meant to be ready to be copied as it is.
+meta:
+  id: # insert an id here. KS currently has no namespaces, so the id MUST be unique within all the repo and this property should be futureproof.
+  title: # insert format human-readable name here
+  application: # Insert the application human-readable name here
+  file-extension:
+    # a YAML list of file extensions here
+  encoding: UTF-8
+  endian: le
```

Comment:
I don't think `encoding` and `endian` belong in the template, especially not pre-filled. Not all formats have a specific encoding or endianness, and if they do it's not always consistent within the format.

Reply:
Oops, forgot to remove them ..